docs/getting-started.md

```diff
@@ -26,7 +26,7 @@ npm init --yes
 
 ### Installation of textlint
    
-You can install `textlint` using npm. We recommend installing `textlint` locally:
+You can install `textlint` using npm. We recommend that you install `textlint` locally by running npm command with `--save-dev` option. This means that npm installs `textlint` in the <your-workspace>/node_modules folder.
```

Comment:
Please enclose &lt;your-workspace&gt;/node_modules with backtick (\`). Without backtick, GitHub thinks &lt;your-workspace&got; as unknown tag and strip it from rendered HTML. Other than that, the modification looks good to me!

Reply:
Thanks! I fixed it.